src/style.css

```diff
@@ -0,0 +1,69 @@
+@font-face {
+  font-family: Avenir-Book;
+  src: url("./fonts/Avenir-Book.ttf");
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url("./fonts/Avenir-Heavy.ttf");
+  font-weight: 800;
+  font-style: normal;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.form {
+  margin: 20px 0;
+}
+
+html {
+  font-family: Avenir-Book, Arial, sans-serif;
+}
+
+.search {
+  font-family: inherit;
+  font-weight: inherit;
+  vertical-align: top;
+  line-height: 100%;
```

Comment:
It is also useless. Nothing changes, if you remove it

Reply:
If you have questions, ask in fe_chat
Linter cant show a mistake because of an absence of a label. There should be another reason. Describe the problem in the chat and add the screenshot.
You should remove unnecessary font-weight and line-height